src/s2_fos/interface.py

```diff
@@ -91,42 +91,42 @@ class PredictorConfig(BaseSettings):
 class S2FOS:
     """
     Interface on to your underlying model.
-
-    This class is instantiated at application startup as a singleton.
-    You should initialize your model inside of it, and implement
-    prediction methods.
-
-    If you specified an artifacts.tar.gz for your model, it will
-    have been extracted to `artifacts_dir`, provided as a constructor
-    arg below.
     """
 
     _config: PredictorConfig
     data_dir: str
 
-    def __init__(self, data_dir: str):
+    def __init__(self, data_dir: str = 'data'):
         self._config = PredictorConfig()
-        self.data_dir = data_dir
-        self._load_model()
+        self.data_dir = os.path.join(PROJECT_ROOT_PATH, data_dir)
```

Comment:
I think this will trip people up.

My advice: 

Default for data_dir is "None"

Then :

```
if data_dir is None:
    self.data_dir = os.path.join(PROJECT_ROOT_PATH, 'data')
else:
    self.data_dir = data_dir
```